Equipment.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Equipment.h"
+#include "Laundromat.h"
+#include "WashingMachine.h"
+#include "Restaurant.h"
+#include "Server.h"
+#include "Speaker.h"
+
+Equipment::Equipment():
+machine(20, 34),
+pipe(24.3f, 43.1f, 9),
+dishWasher(),
+carrier(),
+tweaters()
```

Comment:
None of these 3 need to be in the member initializer list
Their constructors do not take arguments.
- [ ] revise